Dockerfile

```diff
@@ -1,36 +1,20 @@
-# take default image of node boron i.e  node 6.x
-FROM node:8.9 as builder
-RUN npm i -g yarn
-
-# create app directory in container
-RUN mkdir -p /app
+# node image
+FROM node:8.10.0
```

Comment:
2-cents: Could use a lighter base image.
Alpine is about 1/10 the size (~70 Mb vs 700 Mb).